src/ISDisplay.cpp

```diff
@@ -627,8 +628,8 @@ string cInertialSenseDisplay::DataToString(const p_data_t* data)
 	case DID_GPS1_RAW:
 	case DID_GPS2_RAW:
 	case DID_GPS_BASE_RAW:      str = DataToStringRawGPS(d.gpsRaw, data->hdr);          break;
-    case DID_SURVEY_IN:         str = DataToStringSurveyIn(d.surveyIn, data->hdr);      break;
-	case DID_SYS_PARAMS:        str = DataToStringSysParams(d.sysParams, data->hdr);    break;
+    case DID_SURVEY_IN:         str = DataToStringSurveyIn(d.surveyIn, data->hdr);    break;
```

Comment:
You pointed this out to me on a PR, and it seems to still happen randomly, but the code alignment looks funny here.